src/v/cloud_storage/CMakeLists.txt

```diff
@@ -15,3 +16,4 @@ v_cc_library(
     v::rphashing
 )
 add_subdirectory(tests)
+add_subdirectory(cache_demo)
```

Comment:
did you intended to include this directory? the PR doesn't seem to add any content in this subdirectory.

Reply:
Fixed.